tests/local/ictt/ictt_suite_test.go

```diff
@@ -0,0 +1,171 @@
+package ictt_test
+
+import (
+	"context"
+	"os"
+	"testing"
+	"time"
+
+	icttFlows "github.com/ava-labs/teleporter/tests/flows/ictt"
+	"github.com/ava-labs/teleporter/tests/local"
+	"github.com/ava-labs/teleporter/tests/utils"
+	deploymentUtils "github.com/ava-labs/teleporter/utils/deployment-utils"
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+const (
+	teleporterByteCodeFile  = "./out/TeleporterMessenger.sol/TeleporterMessenger.json"
+	warpGenesisTemplateFile = "./tests/utils/warp-genesis-template.json"
+
+	icttLabel              = "ICTT"
+	erc20TokenHomeLabel    = "ERC20TokenHome"
+	erc20TokenRemoteLabel  = "ERC20TokenRemote"
+	nativeTokenHomeLabel   = "NativeTokenHome"
+	nativeTokenRemoteLabel = "NativeTokenRemote"
+	multiHopLabel          = "MultiHop"
+	sendAndCallLabel       = "SendAndCall"
+	registrationLabel      = "Registration"
+	upgradabilityLabel     = "upgradability"
+)
+
+var (
+	LocalNetworkInstance *local.LocalNetwork
+	TeleporterInfo       utils.TeleporterTestInfo
+)
+
+func TestValidatorManager(t *testing.T) {
+	if os.Getenv("RUN_E2E") == "" {
+		t.Skip("Environment variable RUN_E2E not set; skipping E2E tests")
+	}
+
+	RegisterFailHandler(ginkgo.Fail)
+	ginkgo.RunSpecs(t, "Teleporter e2e test")
+}
+
+// Define the Teleporter before and after suite functions.
+var _ = ginkgo.BeforeEach(func() {
```

Comment:
This was `BeforeSuite` and `AfterSuite` before. Is there a reason to set up a new network for each flow?